locales/en-US/formautofill.properties

```diff
@@ -3,26 +3,36 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 preferenceGroupTitle = Form Autofill
-enableProfileAutofill = Enable Profile Autofill
-savedProfiles = Saved Profiles…
+enableAddressAutofill = Autofill addresses
+learnMore = Learn more
+savedAddresses = Saved Addresses…
+enableCreditCardAutofill = Autofill credit cards
+savedCreditCards = Saved Credit Cards…
```

Comment:
As above, if it's a button label, use a clearer ID and/or add a localization comment.

Reply:
Will make it `savedCreditCardsBtnLabel` instead.